docs/docs/photos/features/account/index.md

```diff
@@ -73,6 +73,10 @@ Enable 2FA in `Settings > Security` on mobile or desktop apps.
 
 Learn more about [Passkeys](/photos/features/account/passkeys).
 
+### Session management
```

Comment:
https://ente.io/help/photos/features/account/#multi-device-access feels like a better place to add this.

Reply:
Also, there needs to be "mobile" in "from the web or desktop app"